src/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -135,7 +132,7 @@ public static File getUserRootFolder() {
     final File userHome = new File(System.getProperties().getProperty("user.home"));
     // the default
     File rootDir;
-    if (GameRunner.isMac()) {
+    if (System.getProperties().getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
The Problem is only occuring here... It seems that calling the 'GameRunner' class causes the error...
I think this has to do with the creation of client context, since in the javadoc something about calling gamerunner is mentioned - could be wrong...

TLDR; `GameRunner.isMac()` returns `System.getProperties().getProperty("os.name").toLowerCase().contains("mac")`, errors when calling from here.
Calling `System.getProperties().getProperty("os.name").toLowerCase().contains("mac")` directly works without problems